core/services/validators.go

```diff
@@ -394,6 +396,7 @@ func ValidatedOracleSpec(tomlString string) (offchainreporting.OracleSpec, error
 			ContractConfigTrackerPollInterval:      models.Interval(1 * time.Minute),
 			ContractConfigConfirmations:            uint16(3), // TODO: why a uint16? just forcing casting everywhere
 		},
+		Pipeline: *pipeline.NewTaskDAG(),
```

Comment:
Fix for the issue around not specifying an observationSource in a bootstrap node causing a panic